spec/integration/install_spec.cr

```diff
@@ -419,7 +419,7 @@ describe "install" do
 
     with_shard(metadata) do
       run "shards install"
-      File.touch "shard.lock", Time.utc(1900, 1, 1)
+      File.touch "shard.lock", Time.utc(1901, 12, 14)
```

Comment:
I wonder why such an old time is needed here 🤔 

Can't we just use a date from last year or something like that?

Reply:
Something pre 1970 makes sense I guess in case somebody  boots their system without a hardware clock and it starts from `0` :D

Reply:
Good luck with doing whatever on such machine (think SSL certificates with specific timeframe) ;)